### ISSUE TITLE
Password entries slow on sync between devices

### ISSUE THREAD

**Issue by mossicrue** [bitwarden/clients#12582]
### Steps To Reproduce

Credentials update:

1. Open Bitwarden on macOS client
2. Create a new credential entry
3. Open Bitwarden on iOS app
4. Search for the fresh credential entry
5. Wait something like a couple hours
6. The entry appear correctly on the other devices

Credentials move:

1. Open Bitwarden on iOS app on iPhone A, which user is on an organization
2. Move a credentials from a personal vault to the organization one
3. Open Bitwarden on iOS app on iPhone B, which user is in the same organization
4. Search for the credentials, no credentials available
7. Wait something like a couple hours
8. The entry appear correctly for the other user

### Expected Result

When adding or updating a credentials or moving an existing one from a vault (mine) to another (organization family one) the credentials seems not to be correctly synched when opening from a different device (eg: adding from macOS and check from iOS or Windows).

The vault appear to be synched only after a couple of hours

### Actual Result

Have the vaults synched without waiting this long time. I think seconds may be reasonable.

### Operating System

Windows, macOS

### Operating System Version

Windows 11, macOS 15.1.1, iOS 18.2

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 2024.12.1 SDK 'main (edee180)' Shell 32.1.1 Renderer 128.0.6613.137 Node 20.17.0 Architecture x64

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
